src/main/java/com/fundation/logic/common/MetadataImageExtractor.java

```diff
@@ -101,47 +103,74 @@ public static void width(String width) {
             int freeSpace = 2;
             width = width.substring(initIndex + freeSpace, endIndex);
             searchWidth = width;
+        } else {
+            searchWidth = "All";
         }
     }
 
-    /**
-     * This method returns width image.
-     */
-    public static String getWidth() {
-        return searchWidth;
-    }
-
     /**
      * This method returns metadata image color space.
+     * @return void
      */
-    public static void getColorSpace(String colorSpace) {
-        if ((colorSpace.contains("Color Space"))) {
-            int initIndex = colorSpace.indexOf(":");
-            int endIndex = colorSpace.length();
-            int freeSpace = 2;
-            colorSpace = colorSpace.substring(initIndex + freeSpace, endIndex);
-            searchColorSpace = colorSpace;
+    public static void getColorSpace(String mimeType) {
+        String mimetype2 = mimeType;
```

Comment:
Define more significant name for mimetype2 variable, for example, auxiliarMimetype or another.